src/main/java/ch/njol/skript/expressions/ExprTarget.java

```diff
@@ -153,17 +158,21 @@ public String toString(@Nullable Event event, boolean debug) {
 	/**
 	 * Gets an entity's target.
 	 *
-	 * @param entity The entity to get the target of
+	 * @param entity The entity to get the target of.
 	 * @param type The exact EntityData to find. Can be null for any entity.
-	 * @return The entity's target
+	 * @return The entity's target.
 	 */
 	@Nullable
 	@SuppressWarnings("unchecked")
 	public static <T extends Entity> T getTarget(LivingEntity entity, @Nullable EntityData<T> type) {
 		if (entity instanceof Mob)
 			return ((Mob) entity).getTarget() == null || type != null && !type.isInstance(((Mob) entity).getTarget()) ? null : (T) ((Mob) entity).getTarget();
-
-		RayTraceResult result = entity.rayTraceEntities(SkriptConfig.maxTargetBlockDistance.value());
+		RayTraceResult result;
+		if (!PAPER_RAYTRACE) {
+			result = entity.getWorld().rayTraceEntities(entity.getLocation(), entity.getEyeLocation().toVector(), SkriptConfig.maxTargetBlockDistance.value(), 0.0D);
+		} else {
+			result = entity.rayTraceEntities(SkriptConfig.maxTargetBlockDistance.value(), ignoreBlocks);
+		}
```

Comment:
Cache `SkriptConfig.maxTargetBlockDistance.value()`?

Reply:
The value is cached already

Reply:
It might be better to keep it as is since not caching means the script does not need to be reloaded if the config setting is changed